examples/walkthrough/main.go

```diff
@@ -18,30 +18,41 @@ import (
 
 const defaultExpiration = 7 * 24 * time.Hour
 
-// UploadAndDownloadData uploads the specified data to the specified key in the
-// specified bucket, using the specified Satellite, API key, and passphrase.
-func UploadAndDownloadData(ctx context.Context,
-	accessGrant, bucketName, uploadKey string, dataToUpload []byte) error {
-
+func CreateProjectAndConfirmBucket(ctx context.Context, accessGrant, bucketName string) (*uplink.Project, error) {
```

Comment:
```suggestion
// OpenProjectAndEnsureBucket shows how to open a project and check that the bucket exists and create if it doesn't.
func OpenProjectAndEnsureBucket(ctx context.Context, accessGrant, bucketName string) (*uplink.Project, error) {
```

Note, it's "OpenProject", because it actually doesn't create a new project when it's missing -- just opens a new connection to the project.

Similar reason for the "EnsureBucket" -- "Confirm" would somewhat imply that it would fail if the bucket doesn't exist.